pkgs/development/python-modules/torchaudio/bin.nix

```diff
@@ -0,0 +1,60 @@
+{ lib
+, stdenv
+, buildPythonPackage
+, fetchurl
+, isPy37
+, isPy38
+, isPy39
+, patchelf
+, python
+, pytorch-bin
+}:
+
+let
+  pyVerNoDot = builtins.replaceStrings [ "." ] [ "" ] python.pythonVersion;
+  srcs = import ./binary-hashes.nix version;
+  unsupported = throw "Unsupported system";
+  version = "0.10.0";
+in buildPythonPackage {
+  inherit version;
+
+  pname = "torchaudio";
+
+  format = "wheel";
+
+  src = fetchurl srcs."${stdenv.system}-${pyVerNoDot}" or unsupported;
+
+  disabled = !(isPy37 || isPy38 || isPy39);
```

Comment:
Please use pythonOlder and pythonAtLeast.

Reply:
In this case, it will be `disabled = ! (pythonOlder "3.6" && pythonAtLeast "3.9")`.
When we'd like to pass `3.7, 3.8 and 3.9`, can we specify the version like `3.7 <= version  && version < 4.0`?

Reply:
> In this case, it will be disabled = ! (pythonOlder "3.6" && pythonAtLeast "3.9").

That look already much better.

>  When we'd like to pass 3.7, 3.8 and 3.9, can we specify the version like 3.7 <= version && version < 4.0?

I don't know, probably but the overhead is probably not worth it.

Reply:
I was wrong.
Finally use `disabled =! (pythonAtLeast "3.7" && pythonOlder "3.10")`.